test/elixir_google_scraper_web/features/home_page/view_home_page_test.exs

```diff
@@ -1,9 +1,3 @@
 defmodule ElixirGoogleScraperWeb.HomePage.ViewHomePageTest do
   use ElixirGoogleScraperWeb.FeatureCase
-
-  feature "view home page", %{session: session} do
-    visit(session, Routes.page_path(ElixirGoogleScraperWeb.Endpoint, :index))
-
-    assert_has(session, Query.text("Welcome to Phoenix!"))
-  end
```

Comment:
Should we add feature tests for our new register/login? 🚀

Reply:
Added in https://github.com/junan/elixir_google_scraper/pull/28/commits/fac1bb865c2ea6126ebf74dab313d49435c80a09